data/net/minecraft/world/level/material/FluidState.mapping

```diff
@@ -1,6 +1,27 @@
 CLASS net/minecraft/world/level/material/FluidState
+	METHOD <init> (Lnet/minecraft/world/level/material/Fluid;Lcom/google/common/collect/ImmutableMap;Lcom/mojang/serialization/MapCodec;)V
+		ARG 1 owner
+		ARG 2 values
+		ARG 3 propertiesCodec
+	METHOD animateTick (Lnet/minecraft/world/level/Level;Lnet/minecraft/core/BlockPos;Ljava/util/Random;)V
+		ARG 1 level
+		ARG 2 pos
+		ARG 3 random
+	METHOD getFlow (Lnet/minecraft/world/level/BlockGetter;Lnet/minecraft/core/BlockPos;)Lnet/minecraft/world/phys/Vec3;
+		ARG 1 blockGetter
+		ARG 2 pos
+	METHOD getHeight (Lnet/minecraft/world/level/BlockGetter;Lnet/minecraft/core/BlockPos;)F
+		ARG 1 blockGetter
+		ARG 2 pos
+	METHOD getShape (Lnet/minecraft/world/level/BlockGetter;Lnet/minecraft/core/BlockPos;)Lnet/minecraft/world/phys/shapes/VoxelShape;
+		ARG 1 blockGetter
+		ARG 2 pos
 	METHOD is (Lnet/minecraft/tags/TagKey;)Z
 		ARG 1 tag
+	METHOD is (Lnet/minecraft/world/level/material/Fluid;)Z
+		ARG 1 fluid
+	METHOD isSourceOfType (Lnet/minecraft/world/level/material/Fluid;)Z
+		ARG 1 type
```

Comment:
```suggestion
		ARG 1 fluid
```
Given that this is more akin to other `is` methods on this class, and it's comparing to a fluid. (Fluids are only referred to as the "type" because they're inherited from `StateHolder`, which is used for both block and fluid states.